File: src/net/auoeke/limitless/config/anvil/entry/AnvilNormalizationEntry.java
package net.auoeke.limitless.config.anvil.entry;

import net.auoeke.limitless.config.common.CostDisplay;
import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.EnumHandler;

public class AnvilNormalizationEntry {
    public boolean enabled = true;

    @EnumHandler(option = EnumHandler.EnumDisplayOption.BUTTON)
    public CostDisplay display = CostDisplay.REPLACE;
}

File: src/net/auoeke/limitless/config/anvil/entry/AnvilNormalizationEntry.kt
package net.auoeke.limitless.config.anvil.entry

import me.shedaniel.autoconfig.annotation.ConfigEntry.Gui.EnumHandler
import net.auoeke.limitless.config.common.CostDisplay

class AnvilNormalizationEntry {
    var enabled = true

    @EnumHandler(option = EnumHandler.EnumDisplayOption.BUTTON)
    var display = CostDisplay.REPLACE
}
